public/app/features/dashboard-scene/scene/layout-responsive-grid/ResponsiveGridItemEditor.tsx

```diff
@@ -27,3 +39,16 @@ function GridItemNoDataToggle({ item }: { item: ResponsiveGridItem }) {
 
   return <Switch value={hideWhenNoData} id="hide-when-no-data" onChange={() => item.toggleHideWhenNoData()} />;
 }
+
+function RepeatByOption({ item }: { item: ResponsiveGridItem }) {
+  const { variableName } = item.useState();
+
+  return (
+    <RepeatRowSelect2
+      id="repeat-by-variable-select"
+      sceneContext={item}
+      repeat={variableName}
+      onChange={(value?: string) => item.setRepeatByVariable(value)}
```

Comment:
This also needs `item.performRepeat()`

Reply:
Will add this to the setRepeatByVariable on the grid item